### PR TITLE
Added Google Translate Agent

### PR DESCRIPTION
**Pull request by rrodrigu3z**
## Screenshots

screen shot 2017-03-13 at 10 32 16 pm

screen shot 2017-03-13 at 10 31 50 pm

### CODE REVIEW

**Comment by duboff** [comment]
Hey @rrodrigu3z the `content` field is not very clear. Why are there two fields? Can we just have one called `text`? Documentation doesn't help here either.

**Comment by rrodrigu3z** [comment]
OK, well, the idea is that we could use the agent to translate several fields. By default we could just left one field (`text`). 

Ok I'll improve the doc.

**Comment by duboff** [comment]
Ah, interesting, so each field will be translated? I like that, let's keep it in and just improve the documentation like you say.

But let's only keep one field in the default set.

**Comment by rrodrigu3z** [comment]
Yeah, each field will be translated and optionally merged (true by default) to the incoming payload under the key: `translated_content`.

File: app/models/agents/google_translate_agent.rb [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    form_configurable :content, type: :json, ace: { mode: 'json' }
+    form_configurable :merge, type: :array, values: %w(true false)
+    form_configurable :expected_receive_period_in_days
+
+    def default_options
+      {
+        'to' => 'en',
+        'expected_receive_period_in_days' => 1,
+        'merge' => 'true',
+        'content' => {
```
Comment (duboff) [inline app/models/agents/google_translate_agent.rb:85]: Can we change that to
```js
{
   'comment' => '{{data.comment}}'
}
```

**Comment by rrodrigu3z** [comment]
@duboff Updated

**Comment by duboff** [comment]
LGTM, pls :shipit:

**Pull request merged**